backend/handlers/institution_timeline_handler.py

```diff
@@ -40,10 +40,12 @@ def get(self, user, url_string):
             limit,
             queryPosts)
 
-        array = [post.make(self.request.host) for post in queryPosts]
+        formated_posts = [post.make(self.request.host) for post in queryPosts]
+        visible_posts = [post for post in formated_posts
```

Comment:
Check if you can get the visible posts in the query.

Reply:
Have to do it this way because the is_hidden method uses the make of the post and not the post itself.